### PR TITLE
add elasticsearch-scenario

### PR DESCRIPTION
**Pull request by oatiz**
add elasticsearch test case

### CODE REVIEW

**Comment by ascrutae** [comment]
@oatiz all of test case about elasticsearch senario are failed, Here is the [validate-log](https://github.com/SkywalkingTest/skywalking-agent-testcases/files/2253107/validate-2018-08-02-11-54.log)

**Comment by oatiz** [comment]
@ascrutae pls try again.  fixed a problem without log4j.  I test some cases, they all passed.
<p>

###
|  Version     | Passed | Failed|
|:------------- |:-------:|:-----:|
| 5.3.2  | :heavy_check_mark:||
| 5.3.3  | :heavy_check_mark:||

File: elasticsearch-5.x-scenario/config/expectedData.yaml [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+      - {key: db.instance, value: not null}
+      - {key: db.statement, value: not null}
+      - {key: node.address, value: not null}
+      - {key: es.indices, value: not null}
+      - {key: es.types, value: not null}
+    - operationName: Elasticsearch/actionGet
+      operationId: 0
+      parentSpanId: 0
+      spanId: 2
+      spanLayer: null
```
Comment (ascrutae) [inline elasticsearch-5.x-scenario/config/expectedData.yaml:39]: Why  the`spanLayer` is null?

Comment (oatiz) [reply elasticsearch-5.x-scenario/config/expectedData.yaml:39]: This is a LocalSpan and does not set spanLayer for it.

**Comment by wu-sheng** [comment]
@ascrutae Please run the test again.

**Comment by ascrutae** [comment]
Could you sync the test scenario project? @oatiz

**Comment by wu-sheng** [comment]
Done.

**Comment by oatiz** [comment]
@ascrutae pls run again, set es memory to 256m, here is test report:
<p>

| Case description | Status | Cases|
|:-----|:-----:|:-----:|
|Elasticsearch| **3 passed. 0 failed**| [click me](#elasticsearch) |

## Elasticsearch

###
|  Version     | Passed | Failed|
|:------------- |:-------:|:-----:|
| 5.6.8  | :heavy_check_mark:||
| 5.6.9  | :heavy_check_mark:||
| 5.6.10  | :heavy_check_mark:||

**Comment by oatiz** [comment]
@ascrutae  pls run again, change es discovery type, this is test report (12c24g):
<p>

## Cases List

| Case description | Status | Cases|
|:-----|:-----:|:-----:|
|Elasticsearch| **17 passed. 8 failed**| [click me](#elasticsearch) |

## Elasticsearch

###
|  Version     | Passed | Failed|
|:------------- |:-------:|:-----:|
| 5.4.2  | :heavy_check_mark:||
| 5.6.9  | :heavy_check_mark:||
| 5.3.1  | |:heavy_check_mark:|
| 5.4.3  | :heavy_check_mark:||
| 5.6.6  | :heavy_check_mark:||
| 5.5.1  | :heavy_check_mark:||
| 5.2.1  | |:heavy_check_mark:|
| 5.6.0  | :heavy_check_mark:||
| 5.5.3  | :heavy_check_mark:||
| 5.6.8  | :heavy_check_mark:||
| 5.5.0  | :heavy_check_mark:||
| 5.6.5  | |:heavy_check_mark:|
| 5.3.2  | |:heavy_check_mark:|
| 5.3.3  | |:heavy_check_mark:|
| 5.5.2  | :heavy_check_mark:||
| 5.6.1  | :heavy_check_mark:||
| 5.6.7  | :heavy_check_mark:||
| 5.3.0  | |:heavy_check_mark:|
| 5.6.3  | :heavy_check_mark:||
| 5.2.0  | |:heavy_check_mark:|
| 5.6.4  | |:heavy_check_mark:|
| 5.4.1  | :heavy_check_mark:||
| 5.6.2  | :heavy_check_mark:||
| 5.4.0  | :heavy_check_mark:||
| 5.6.10  | :heavy_check_mark:||

**Review by ascrutae — APPROVED** [review]

**Pull request merged**